### PR TITLE
updated the README insights

### PR DESCRIPTION
**Pull request by ZainabSayyed**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,10 +34,10 @@ You can setup this fully functional reference architecture (which includes test
 [![Launch Cloud Shell](https://azurecomcdn.azureedge.net/mediahandler/acomblog/media/Default/blog/launchcloudshell.png "Launch Cloud Shell")](https://shell.azure.com/bash)
 1) Download the OEA framework setup script and framework assets to your Azure clouddrive \
 `cd clouddrive`\
-`wget https://github.com/microsoft/OpenEduAnalytics/releases/download/v0.8rc1/OEA_v0.8rc1.zip`\
-`unzip ./OEA_v0.8rc1.zip`
+`wget https://github.com/microsoft/OpenEduAnalytics/releases/download/v0.8rc1/OEA_v0.8.zip`\
```
Comment (shreenidhira) [inline README.md]: Shouldn't this be 0.8 as well? i.e https://github.com/microsoft/OpenEduAnalytics/releases/download/**v0.8**/OEA_v0.8.zip

File: modules/module_catalog/Microsoft_Education_Insights/README.md [hunk 2]
```diff
@@ -50,10 +50,10 @@ All the steps outlined below are applicable to deployment of this module with pr
 [![Launch Cloud Shell](https://azurecomcdn.azureedge.net/mediahandler/acomblog/media/Default/blog/launchcloudshell.png "Launch Cloud Shell")](https://shell.azure.com/bash)
     * Download the module release to your Azure clouddrive \
 `cd clouddrive`\
-`wget https://github.com/microsoft/OpenEduAnalytics/releases/download/v0.2rc1/module_Microsoft_Education_Insights_v0.2rc1.zip`\
-`unzip ./module_Microsoft_Education_Insights_v0.2rc1.zip`
+`wget https://github.com/microsoft/OpenEduAnalytics/releases/download/v0.2rc1/module_Microsoft_Education_Insights_v0.2.zip`\
```
Comment (shreenidhira) [inline modules/module_catalog/Microsoft_Education_Insights/README.md]: Should be v0.2 instead of v0.2rc1.

**Review by dastewa — APPROVED** [review]
Lgtm

**Comment by ref779** [comment]
how can I know you guys randomly don't invase my pc again? And BTW did ya boys tried to open my webcam?

**Pull request merged**